site/docs/extensions/filter-control.md

```diff
@@ -6,8 +6,6 @@ group: extensions
 toc: true
 ---
 
-Dependence if you use the datepicker option: [bootstrap-datepicker](https://github.com/eternicode/bootstrap-datepicker) v1.4.0
```

Comment:
We should add info that the filter-control maybe needs the multipleSelect lib as dependency.
I saw that you already wrote it on the options, but the top of the page is the first place a develop sees.

Reply:
Got it

Reply:
Added